utils/src/main/scala/za/co/absa/enceladus/utils/transformations/ArrayTransformations.scala

```diff
@@ -144,10 +153,12 @@ object ArrayTransformations {
     val arraySchema = ArrayType.apply(arrayChildSchema, origArraySchema.containsNull)
 
     spark.udf.register(s"${groupField}_handleNullAndEmpty", new UDF2[Int, Seq[Row], Seq[Row]] {
-      override def call(t1: Int, t2: Seq[Row]) = {
-        if (t1 == -1) null
+      override def call(t1: Int, t2: Seq[Row]): Seq[Row] = {
+        // scalastyle:off if.brace
```

Comment:
Alternatively, consider putting braces maybe. This use case doesn't look so special that it needs scala style warning suppression.